.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  build:
+    name: build
+    runs-on: ubuntu-18.04
```

Comment:
any reason for now using `ubuntu-20.04` here? see https://github.com/actions/virtual-environments/

Reply:
I believe Brad originally picked 18.04 (and I stuck with it), because that gave us more flexibility with our debian packages. That is, if we built on 18.04, the `.deb` worked on 18.04, 20.04, and 20.10 with no modifications. But if we built on a newer version, we got no backwards compatibility.